### ISSUE TITLE
Why does the core network include only 20 transcription factors?

### ISSUE THREAD

**Issue by ShuhuaGao** [swoodhouse/SCNS-GUI#8]
Hi, Dr. Steven Woodhouse,

Much appreciation of your excellent work. I am very interested in the tool SCNS for network reconstruction. However, I have a question about your paper "Decoding the regulatory network of early blood development from single-cell gene expression measurements", which proposes SCNS. (In fact, I sent an email to your coauthor Dr. Jasmin Fisher for inquiry, but got no response after a week.)

In short, in Figure 3 of the above mentioned paper, the core network contains 20 transcription factors. However, in the original data, 33 transcription factors are profiled. How did you exclude the 13 factors from the reconstructed network? Is there some criterion?

It would be best if you can drop me a mail on this topic to nus.gao.shuhua@gmail.com. Thanks a lot.

Best regards,
Shuhua
